.github/workflows/build.yml

```diff
@@ -130,13 +136,16 @@ jobs:
           toolchain: "1.71"
 
       # zlib-rs is optional and requires a later MSRV
-      - run: cargo check --locked --lib $(admin/all-features-except zlib ${{ env.MAIN_CRATE }}) -p ${{ env.MAIN_CRATE }}
+      - run: cargo check --locked --lib ${{ env.ALL_FEATURES_EXCEPT_UNSAFE_AND_ZLIB }} -p ${{ env.MAIN_CRATE }}
```

Comment:
may be cleaner to get rid of some inner spaces in general (throughout these updates), something like this, if possible (showing as diff, with leading spaces removed):

```diff
-  - run: cargo check --locked --lib ${{ env.ALL_FEATURES_EXCEPT_UNSAFE_AND_ZLIB }} -p ${{ env.MAIN_CRATE }}
+  - run: cargo check --locked --lib ${{env.ALL_FEATURES_EXCEPT_UNSAFE_AND_ZLIB}} -p ${{env.MAIN_CRATE}}
```